receiver/kubeletstatsreceiver/README.md

```diff
@@ -218,6 +218,41 @@ receivers:
       - pod
 ```
 
+### Collect k8s.container.cpu.node_limit_utilization as ratio of total node's capacity
+
+In order to calculate the `k8s.container.cpu.node_limit_utilization` metric, the information of the node's capacity
+must be retrieved from the k8s API. In this, the `k8s_api_config` needs to be set.
+In addition, the node name must be identified properly. When the collector runs as Daemonset,
+the receiver can get this information automatically by the `K8S_NODE_NAME` env var:
+
+```yaml
+env:
+  - name: K8S_NODE_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: spec.nodeName
+```
+
+When the receiver runs as a Deployment (along with a receiver creator for example)
+then for each receiver instance the `node` setting must be set:
+
+```yaml
+receiver_creator/kubelet:
+    watch_observers: [k8s_observer]
+    receivers:
+      kubeletstats:
+        rule: type == "k8s.node"
+        config:
+          auth_type: serviceAccount
+          collection_interval: 10s
+          k8s_api_config:
+            auth_type: serviceAccount
+          endpoint: '`endpoint`:`kubelet_endpoint_port`'
+          node: '`name`'
```

Comment:
Let's reference the env var you describe earlier

Reply:
As I explain at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32295#discussion_r1603980927, in  the Deployment case we discover the target Nodes through the `k8s_observer`. Then for each discovered Node we want to use its actual name to set the `node` setting for its respective receiver generated configuration.
If we use the `K8S_NODE_NAME` to set the `node` setting all the generated receivers' configurations will be pointing to the same node. I think that's not we want here.

Reply:
Is \`name\` a keyword? It isn't clear to me how this works in `deployment` mode.  With this config it looks like `NodeName` would get set to \`name\`, which I wouldn't expect to match anything in the scraped node details.

Reply:
Yeap, as per https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md#kubernetes-node,  \`name\` is a variable here that will be populated by the `receiver_creator` for each discovered Node accordingly.

Reply:
Oh I didn't realize this was a receivercreator example.  How would someone configure the kubeletstatsreceiver directly to use this metric in deployment mode?

Reply:
Using a dynamic configuration through the receivercreator is what I would suggest anyway since the `endpoint` setting needs to be set for every single node right?
If users want to manually set the `endpoint`s for each node of the cluster then they would need to also set the `node` as well for each receiver's instance. 
Is there any other way that we would do that today?

Reply:
I don't believe we support the kubeletstatsreceiver being deployed as a Deployment on clusters with more than 1 node, but I believe it would technically work, you'd just end up with duplicate data or missing some node metrics.  This is because if `endpoint` is not set we default to using the hostname.

Should we do something similar for the new `node` field? This could smooth the use of the new metric for either deployment mode.

I also say we don't need to worry about documenting anything about deployment directly, I was getting mixed up with the k8sattributesprocessor.

Reply:
> Should we do something similar for the new node field? This could smooth the use of the new metric for either deployment mode.

I originally had added this kind of fallback relying on a reasonable default, but I removed that based on https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32295#discussion_r1600674347.

In any case, I think that for multi-node clusters the Deployment mode can work using the `receivercreator` as the respective examples already indicate at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md#examples, but we can leave it out for now from the docs and revisit if we see the need.

I removed the section ([8b8a95c](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32295/commits/8b8a95cc19525cb70d2a664de8e8bce79f88a584)), please take another look.

Reply:
I think we could do the same strategy we use for endpoint, which is using the hostname, but we could add that enhancement in the future.